src/domain/child/modal/ChildFormModal.tsx

```diff
@@ -198,6 +213,16 @@ const ChildFormModal: React.FunctionComponent<ChildFormModalProps> = ({
                 >
                   {t('child.form.modal.add.label')}
```

Comment:
This is a bug from before :) It shouldn't show the add label when you are editing!

Reply:
Fixed